### ISSUE TITLE
Possible to make it work with just the thumbImage? 

### ISSUE THREAD

**Issue by sabeehshah** [wittlock/ngx-image-zoom#15]
I need this for about 500 products and each product has 5 different images. If i need two copies of each image to link then that is a lot of images. Is there a way to have it work just the same by having only 1 image? or is there a magical way of including the same image with a different size in the code?

**Comment by wittlock** [comment]
The attribute fullImage is optional and can be skipped, combined with a magnification setting bigger than 1 you can let the users zoom in above 100%

But if it's a page with a lot of products I'm guessing you have a highres version of the image. In its current shape the image will be as big as the image using as thumbImage. There's an improvement request already in #6 requesting a setting to change the size of things instead of just being 100% width/height of thumbImage.

If you let the browser scale the images down though it may not look as good as if you make real thumbnails.

Bottom line, currently the only way to work with only one image right now is to zoom from 100% of thumbImage size to >100% of thumbImage size. I think what you probably are asking for is what #6 is requesting. If that's the case, add your vote on that one (to help me and others find good pics for next feature) or submit a pull request. :)

**Comment by wittlock** [comment]
No follow up so I'm going to close this as resolved. Feel free to report back if there's more information to be shared and it can be reopened.

**Comment by siba1985** [comment]
.ngxImageZoomThumbnail{
    width: 580px!important;
    height: 580px!important;
}
In main style.scss
